.goreleaser.yml

```
@@ -130,6 +130,35 @@ archives:
       - config.json
       - neighbors.json
 
+# Additional packages (i.e. deb)
+nfpms:
+-
+  id: nfpm
+  package_name: hornet
+  file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
```

We should use the dpkg name convention:
https://manpages.debian.org/buster/dpkg-dev/dpkg-name.1.en.html

```suggestion
  file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Arch }}"
```